Masquerade4997Library/src/main/java/Library4997/MasqControlSystems/MasqPID/MasqPIDController.java

```diff
@@ -70,4 +79,17 @@ public void setKd(double kd) {
     public MasqClock getClock() {
         return clock;
     }
-}
+
+    public double getKp() {
+        return kp;
+    }
+
+    public double getKi() {
+        return ki;
+    }
+
+    public double getKd() {
+        return kd;
+    }
```

Comment:
Did you pull my changes? This should be an array output. Either pull or take a look at whats on github